tox.ini

```diff
@@ -29,7 +30,7 @@ deps =
     cmake
 
 commands =
-    pip wheel python/equistore-core --no-build-isolation --no-deps --check-build-dependencies --wheel-dir {envtmpdir}/dist
+    pip wheel python/equistore-core --no-deps --no-build-isolation --check-build-dependencies --wheel-dir {envtmpdir}/dist
```

Comment:
is the order important?

Reply:
not really, I just changed this back and forth a lot of time ^^. I'll check to use the same order for all commands in this file

Reply:
I've added a tox variable to hold all of these flags